source/types/itemList.ts

```diff
@@ -38,3 +38,11 @@ export class ItemList<TItemType extends IItem> {
 		return this.items;
 	}
 }
+
+export class SortedItemList<TItemType extends IItem> extends ItemList<TItemType> {
+	setItems(items: TItemType[]): void {
+		super.setItems(items.sort((i1, i2) => {
+			return i1.display.localeCompare(i2.display);
```

Comment:
That'd be cleaner but unfortunately sort takes a function that should return -1, 0, or 1 depending on whether `i1<i2`, `i1 === i2`, or `i1>i2` respectively. This seemed a better way than doing all the if checks manually.